src/components/Header/BackLink/BackLink.css

```diff
@@ -0,0 +1,17 @@
+.back {
```

Comment:
for some custom component i would suggest you to use wrapper and inside it all styles. Because now  if the classNames somehow coincide, the styles can be overwritten